runtime_magic.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2022, The Garble Authors.
```

Comment:
this file is only 45 lines, and we already have the small `runtime_strip.go` at 145 lines. merge both into `runtime_patch.go`?